code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -123,11 +123,18 @@
 	list_reagents = list("epinephrine" = 30, "weak_omnizine" = 30, "salglu_solution" = 30)
 
 /obj/item/reagent_containers/hypospray/combat/nanites
-	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with expensive medical nanites for rapid healing."
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with a cocktail of experimental combat drugs and <b>extremely</b> expensive medical nanomachines. Capable of healing almost any injury in just a few seconds."
 	icon_state = "nanites_hypo"
 	volume = 100
 	list_reagents = list("nanites" = 100)
 
+/obj/item/reagent_containers/hypospray/combat/syndicate_nanites
+	name = "medical nanite injector"
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with expensive medical nanomachines for rapid field stabilization."
+	volume = 100
+	possible_transfer_amounts = list(1,2,3,4,5,10,15,20,25,30) // i want to let them to inject more/less depending on the situation, while keeping the flags the base combat has
```

Comment:
```suggestion
```
Don't need it, base hypo has it and there's no overrides.

Reply:
`hypospray/combat` indeed overrides it
`possible_transfer_amounts = null`

Reply:
changed the base type to have it